conans/client/graph/python_requires.py

```diff
@@ -58,7 +59,11 @@ def _look_for_require(self, require):
                 # Will register also the aliased
                 python_require = self._look_for_require(conanfile.alias)
             else:
-                python_require = PythonRequire(new_ref, module, conanfile)
+                package_layout = self._proxy._cache.package_layout(new_ref)  # FIXME: access private
```

Comment:
use conanfile.short_paths, unlikely but still possible that ``exports_source`` is shortened